convert2rhel/subscription.py

```diff
@@ -81,8 +81,16 @@ def unregister_system():
         loggerinst.info("Skipping due to the use of --keep-rhsm.")
         return
 
-    submgr_installed = pkghandler.get_installed_pkg_objects("subscription-manager")
-    if not submgr_installed:
+    # We are calling run_subprocess with rpm here because of a bug in
+    # Oracle/CentOS Linux 7 in which the process always exit with 1 in case of a
+    # rollback when KeyboardInterrupt is raised, so, to avoid many changes and
+    # different conditionals, we are doing a simple call for rpm to verify if
+    # subscription-manager is installed on the system.  This is the current line
+    # on `rpm` that causes the process to exit with any interaction with the yum
+    # library
```

Comment:
```suggestion
    # We are calling run_subprocess with rpm here because of a bug in
    # Oracle/CentOS Linux 7 in which the process always exits with 1 in case of a
    # rollback when KeyboardInterrupt is raised.  To avoid many changes and
    # different conditionals to handle that, we are doing a simple call to rpm to verify if
    # subscription-manager is installed on the system.  This is the current line
    # in `rpm` that causes the process to exit with any interaction with the yum
    # library
```